### DISCUSSION TITLE
Open same Group/Workspace on both screens

### DISCUSSION THREAD

**Discussion by nullbyto** [qtile/qtile#3731] [Q&A]
AFAIK to _display_ same workspace twice is not possible.
What I'm looking for is how to open the same `group/workspace` in both screens which just **expands** the `workspace`, not duplicate it.

For example:
- Workspace `1` has only terminals, which some are on the right screen and some others on the second screens.

Is this possible?

What now happens is when the `screen 1` is on workspace `1` and `screen 2` is on workspace `2` and i try to switch to the other screen's workspace they interchange.

**Comment by elParaguayo** [comment]
Off the top of my head, I'd say that it's almost certainly _possible_ but it would need some very ugly hacking and may break other things!

Currently, layouts are confined to the dimensions of the screen. To extend it onto another screen you'd need to do something like one of these:
 - tell the group that it's got a bigger area to draw into (and the other group has a smaller space/no space)
 - redefine the screens dimensions so one screen extends onto the other (this is probably "easier" but would also impact the bar)
 - Have some means to toggle either of these

**Reply by m-col** [reply]
Perhaps it's already doable using fake screens? http://docs.qtile.org/en/stable/manual/config/screens.html#fake-screens

**Reply by elParaguayo** [reply]
Yes. But either way, you're changing dimensions via some toggle.

**Reply by m-col** [reply]
Ah yes, it might be straight forward to always have a 'single screen' with 2 physical monitors but swapping between that and the usual approach would require a config reload at the very least

**Reply by nullbyto** [reply]
Sounds good.

I was thinking if the second monitor with a different `screen` would have the same group names as the first but actually are others, this would replicate the same situation. As if it extends it.
Correct me if i'm wrong.

**Reply by elParaguayo** [reply]
Not entirely sure I follow what you're saying. However, you don't define groups per `Screen`; the list of all groups is available to any screen.

**Reply by nullbyto** [reply]
I found #3035, which actually has a good solution. This is what I mean, but with the same `group` names (and optionally same keybindings on screen of focus. Depending if one wants to switch workspace on all screens together.) showing on the `GroupBox` for each screen.

For instance:
Two monitors both have groups `1`, `2`, `3`. On Monitor `A` group `1` is not actually the same as group `1` on Monitor B.
